components/ApiCollection/index.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+
+export default function ApiCollection({ data, error }) {
+  const [id, setId] = useState(0);
+  const [comicsInfo, setComicsInfo] = useState([]);
+
+  function handlePrevComic() {
+    setId(data.prevId);
+  }
+
+  function handleNextComic() {
+    setId(data.nextId);
+  }
+
+  function handleAddToOwnCollection(id) {
+    setComicsInfo((prevComicsInfo) => {
+      const info = prevComicsInfo.find((info) => info.id === id);
+      if (info) {
+        return prevComicsInfo.map((info) =>
+          info.id === id
+            ? { ...info, isInOwnCollection: !info.isInOwnCollection }
+            : info
+        );
+      }
+      return [...prevComicsInfo, { id: id, isInOwnCollection: true }];
+    });
+  }
+
+  if (!data) {
+    return <h1>Loading...</h1>;
+  }
+
```

Comment:
i get a little confused by state id and id on this page...

Reply:
me too